zml/aio/torch/parser.zig

```diff
@@ -418,16 +418,16 @@ test "Read pickle (simple)" {
                 try testing.expect(key == .string);
```

Comment:
did you try "testing.expectEqualsDeep" ? does it work with this struct ?

Reply:
Was able to get that to work, will be fixed in the next commit!